server/support_packet.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package main
+
+import (
+	"path"
+
+	"github.com/pkg/errors"
+	"gopkg.in/yaml.v3"
+
+	"github.com/mattermost/mattermost/server/public/model"
+	"github.com/mattermost/mattermost/server/public/plugin"
+)
+
+type SupportPacket struct {
+	Version string `yaml:"version"`
+	// The total number calls.
+	TotalCalls int64 `yaml:"total_calls"`
+	// The total number of active calls.
+	TotalActiveCalls int64 `yaml:"total_active_calls"`
+	// The total number of active sessions.
+	TotalActiveSessions int64 `yaml:"total_active_sessions"`
+	// The average calls duration in seconds.
+	AvgDuration int64 `yaml:"avg_duration"`
+	// The average peak number of participants in calls.
+	AvgParticipants int64 `yaml:"avg_participants"`
```

Comment:
Not overly in love with the repetition here. Wondering if we could embed `CallsStats` or perhaps create a shared [public](https://github.com/mattermost/mattermost-plugin-calls/tree/main/server/public) subtype.

Reply:
I'm not so sure if we should combine these two data structs. While they share common data, they don't share common uses. Is it worth avoiding a copy by coupling them?